spark_connect_labextension/config.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+EXTENSION_ID = 'spark-connect-labextension'
+
+SPARK_HOME = os.getenv('SPARK_HOME')
+SPARK_CONNECT_PACKAGE = os.getenv('SPARK_CONNECT_PACKAGE', 'org.apache.spark:spark-connect_2.12:3.4.0')
+SPARK_CONNECT_PORT = int(os.getenv('SPARK_CONNECT_PORT', '15002'))
+
+SPARK_CLUSTER_NAME = os.getenv('SPARK_CLUSTER_NAME')
```

Comment:
Would it be here the place to have options for the location of bundles, spark options, clusters, error suggestions ?

Reply:
Bundles, spark options, clusters, and error suggestions are configured in the extension config files. This `config.py` file is only for retrieving config from environment variables

Reply:
Yes but I mean configuring here the variables that point to the location of those extension config files, which load some environment setting (as you do here e.g. for SPARK_CLUSTER_NAME). Because in SWAN, we will need variables that point to the locations on CVMFS where bundles, spark_options and clusters are located. For error_suggestions, I'm not sure if the extension should just include some common Spark error pattern matching configuration file (so this one does not need to be on CVMFS and therefore does not need an env variable that points to it).

Reply:
So for the bundles and configs, it is defined on the Jupyter config file, under `spark_options_from_file` option. Since, in my opinion, retrieving bundle from CVMFS is CERN-specific, I think it's better to not put the env here. 

Instead, we could put it on the SWAN singleuser config file, as seen here: https://gitlab.cern.ch/hilmy/swan-spark-connect-labextension/-/blob/master/jupyter_notebook_config.py#L45. We could use `os.getenv()` in that file if the path needs to be retrieved from env.

Reply:
Ok so the logical division of options is: these options here in `config.py` are parameters for the server extension to create the spark connect driver, and the options in `jupyter_notebook_config.py` are parameters for the frontend to know which bundles, spark_options and clusters to show. All fine then.

For the errors we can just have some preconfigured error suggestions in `jupyter_notebook_config.py`.